unittests/AST/ASTImporterTest.cpp

```diff
@@ -3173,6 +3173,105 @@ TEST_P(
                     .match(ToTU, classTemplateSpecializationDecl()));
 }
 
+class ImportImplicitMethods : public ASTImporterTestBase {
+public:
+  static constexpr auto DefaultCode = R"(
+      struct A { int x; };
+      A&& fa() { return static_cast<A&&>(A()); }
+      void f() {
+        A a;
+        A a1(a);
+        A a2(fa());
+        a = a1;
+        a = fa();
+        a.~A();
+      })";
+
+  template<typename MatcherType>
+  void test(const MatcherType &MethodMatcher,
+      const char *Code = DefaultCode, bool CheckNoImport = false) {
```

Comment:
Why don't we just pass the `ExpectedCount` as a parameter? Default could be `1`.
`CheckNoImport` is an opaque name for me, I had to understand the whole `test` code before knowing it's real meaning.